consensus/ibft/ibft.go

```diff
@@ -1249,13 +1249,31 @@ func (i *Ibft) VerifyHeader(parent, header *types.Header) error {
 	}
 
 	// verify the committed seals
-	if err := verifyCommitedFields(snap, header); err != nil {
+	if err := verifyCommitedFields(snap, header, i.quorumSize(header.Number)); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func (i *Ibft) quorumSize(blockNumber uint64) QuorumImplementation {
```

Comment:
Can you please add some comment as to what this method does? 🙏

Reply:
[Added description](https://github.com/0xPolygon/polygon-edge/pull/549/commits/22e04571a284c2f404fd0bdc1c47a8f09e458d54)